tapioca/tapioca.py

```diff
@@ -261,8 +261,19 @@ def _reached_max_limits(self, page_count, item_count, max_pages,
         reached_item_limit = max_items is not None and max_items <= item_count
         return reached_page_limit or reached_item_limit
 
-    def pages(self, max_pages=None, max_items=None, **kwargs):
-        executor = self
+    def pages(self, max_pages=None, max_items=None, params=None, **kwargs):
+        if self._response is not None:
+            if params is not None:
+                raise Exception("Since you're paging after the first .get call, "
```

Comment:
Just improving the message:
`You cannot pass new params after the first request is done. Use the ".get" method to pass parameters`